File: 240718/hashset 기본/hashset-basic.kt
import java.io.*
import kotlin.collections.*

val set = HashSet<Int>()

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val num = br.readLine().toInt()
    for (i in 0 until num) {
        val command =
            br
                .readLine()
                .trim()
                .split(" ")
                .toTypedArray()
        when (command[0]) {
            "find" -> {
                println(set.contains(command[1].toInt()))
            }
            "add" -> set.add(command[1].toInt())
            "remove" -> set.remove(command[1].toInt())
        }
    }
}

File: 240718/hashset 기본/hashset-basic.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split("\n");
const commands = input.slice(1).map(i => i.trim().split(" "));

let s = new Set();

commands.forEach(com => {
    switch(com[0]) {
        case "find":
            console.log(s.has(Number(com[1])));
            break;
        case "add":
            s.add(Number(com[1]));
            break;
        case "remove":
            s.delete(Number(com[1]));
            break;
    }
})